test/quantization/test_quantize_jit.py

```diff
@@ -3026,6 +3026,36 @@ def test_conv(self):
                 inplace=False)
             self.assertEqual(model_quantized(self.img_data_2d[0][0]), result_eager)
 
+    @override_qengines
+    def test_conv_transpose(self):
+        r"""Compare the result of quantizing conv_transpose layer in
+        eager mode and graph mode
+        """
+        if qengine_is_qnnpack():  # Currently only qnnpack is supported
+            # eager mode
+            annotated_conv_model = AnnotatedConvModel(
+                torch.backends.quantized.engine, transposed=True).eval()
+            conv_model = ConvModel(transposed=True).eval()
+            # copy the weight from eager mode so that we can
+            # compare the result of the two quantized models later
+            conv_model.conv.weight = torch.nn.Parameter(annotated_conv_model.conv.weight.detach())
+            model_eager = quantize(annotated_conv_model, test_only_eval_fn, self.img_data_2d)
+            qconfig_dict = {'': get_default_qconfig(torch.backends.quantized.engine)}
+            model_traced = torch.jit.trace(conv_model, self.img_data_2d[0][0])
+            model_script = torch.jit.script(conv_model)
+            result_eager = model_eager(self.img_data_2d[0][0])
+            for model_under_test in [model_traced, model_script]:
+                model_quantized = quantize_jit(
+                    model_under_test,
+                    qconfig_dict,
+                    test_only_eval_fn,
+                    [self.img_data_2d],
+                    inplace=False)
+                self.assertEqual(model_quantized(self.img_data_2d[0][0]),
+                                 result_eager,
+                                 rtol=1e-2,
+                                 atol=1e-2)
+
```

Comment:
@jerryzh168 For some reason there is no exact match -- 1e-2 works though :/

Reply:
can you show is it the traced model or scripted model that is failing?

Reply:
It's failing for both trace model and the script model

Reply:
Oh I think the graph is not even replaced with the quantized versions -- I might have missed something

Reply:
yeah, please print the graphs

Reply:
> yeah, please print the graphs

```console
Eager model AnnotatedConvModel(
  (conv): QuantizedConvTranpose2d(3, 5, kernel_size=(3, 3), stride=(1, 1), scale=0.004468684550374746, zero_point=151)
  (quant): Quantize(scale=tensor([0.0039]), zero_point=tensor([0]), dtype=torch.quint8)
  (dequant): DeQuantize()
)

MUD graph graph(%self : __torch__.torch.testing._internal.common_quantization.___torch_mangle_3.ConvModel,
      %x.1 : Tensor):
  %4 : None = prim::Constant()
  %2 : __torch__.torch.nn.modules.conv.___torch_mangle_2.ConvTranspose2d = prim::GetAttr[name="conv"](%self)
  %x.3 : Tensor = prim::CallMethod[name="forward"](%2, %x.1, %4) # /home/zafar/Git/pytorch-dev/pytorch-convtranspose-new/torch/testing/_internal/common_quantization.py:594:12
  return (%x.3)

quantized MUD graph graph(%self : __torch__.torch.testing._internal.common_quantization.___torch_mangle_18.ConvModel,
      %x.1 : Tensor):
  %5 : int = prim::Constant[value=1]() # /home/zafar/Git/pytorch-dev/pytorch-convtranspose-new/torch/nn/modules/conv.py:909:66
  %4 : int[] = prim::Constant[value=[0, 0]]()
  %3 : int[] = prim::Constant[value=[1, 1]]()
  %self.conv.bias : None = prim::Constant()
  %self.conv.weight : Float(3:45, 5:9, 3:3, 3:1, requires_grad=0, device=cpu) = prim::Constant[value=<Tensor>]()
  %x.3 : Tensor = aten::conv_transpose2d(%x.1, %self.conv.weight, %self.conv.bias, %3, %4, %4, %5, %3) # /home/zafar/Git/pytorch-dev/pytorch-convtranspose-new/torch/nn/modules/conv.py:911:15
  return (%x.3)
```

Reply:
#resolved Thanks Jerry!